packages/client/machines/appMusicPlaylistsMachine.ts

```diff
@@ -400,6 +482,47 @@ export function createAppMusicPlaylistsMachine({
                     },
                     ///
 
+                    //Get context
+                    MPE_GET_CONTEXT: {
+                        actions: forwardTo('socketConnection'),
+                    },
+
+                    RECEIVED_MPE_GET_CONTEXT_FAIL_CALLBACK: {
+                        actions: [
+                            'displayGetContextFailureToast',
+                            'navigateToMpeRoomsSearchScreen',
+                            stop((_, event) =>
+                                getPlaylistMachineActorName(event.roomID),
+                            ),
+                            send(
+                                (_, { roomID }) =>
+                                    appMusicPlaylistsModel.events.REMOVE_ACTOR_FROM_PLAYLIST_LIST(
+                                        {
+                                            roomID,
+                                        },
+                                    ),
+                                {
+                                    delay: 100,
+                                },
+                            ),
```

Comment:
I think we should not rely on an arbitrary timeout.
Replacing error throwing in `usePlaylist` hook with a return type of either the actor or undefined could be a solution.

Reply:
Yes will look forward